README.md

```diff
@@ -1,2 +1,56 @@
 # AMT-CONFERENCE
-Africa must think conference is aimed at redefining the future of Africa by transforming the education system from the theoretical approach to that which appreciates practical Entrepreneurship skills.
+
+Africa must think conference is aimed at redefining the future of Africa by transforming the education system from the theoretical approach to that which appreciates practical Entrepreneurship skills. Here is the link video [Loom Video](https://www.loom.com/share/08ebad94e4774b8491249d70951e9488) explaining how it works. For the live conference [Click here](https://evans22j.github.io/AMT-CONFERENCE/) to see the version of the conference page.
+
+Below are the screenshorts links for:
+
+Desktop Version.
+
+[screenshot](./images/icons/screen-desk.png)
+
+Mobile Version
+
+[screenshot](./images/icons/mobile.png)
+
```

Comment:
- [ ] You have added the screenshots as links. I would recommend adding them as images to your Readme. To do that try to add `!` symbol before `[screenshot]`
```md
Desktop Version.

![screenshot](./images/icons/screen-desk.png)

Mobile Version

![screenshot](./images/icons/mobile.png)
```

Reply:
DONE